site/content/docs/user/quick-start.md

```diff
@@ -370,6 +370,22 @@ networking:
   ipFamily: ipv6
 ```
 
+#### Dual Stack clusters
+You can run dual stack clusters using `kind`, on kubernetes versions +1.21
+but first you need to [enable ipv6 in your docker daemon][docker enable ipv6].
+
+```yaml
+# a dual-stack cluster
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  ipFamily: dual
+nodes:
```

Comment:
todo (follow-up): just show the minimal config relevant to get a dualstack cluster.

Also consider moving this entirely to the the configuration page, as we have a better configuration guide over there and this page is way too disorganized and lengthy already.